test/unit/sites/infibeam-test.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const InfibeamSite = require('../../../lib/sites/infibeam');
+const cheerio = require('cheerio');
+const siteUtils = require('../../../lib/site-utils');
+
+const VALID_URI = 'http://www.infibeam.com/Home_Entertainment/sansui-sjx22fb-full-hd-led-tv/P-hoen-68091831042-cat-z.html#variantId=P-hoen-60492354794';
+const INVALID_URI = 'http://www.bad.com/123/product';
+
+describe('The Infibeam Site', () => {
+  it('should exist', () => {
+    expect(InfibeamSite).toBeDefined();
+  });
+
+  describe('isSite() function', () => {
+    it('should return true for a correct site', () => {
+      expect(InfibeamSite.isSite(VALID_URI)).toBeTruthy();
+    });
+
+    it('should return false for a bad site', () => {
+      expect(InfibeamSite.isSite(INVALID_URI)).toBeFalsy();
+    });
+  });
+
+  it('should throw an exception trying to create a new site with an incorrect uri', () => {
+    expect(() => {
+      /* eslint-disable no-new */
+      new InfibeamSite(INVALID_URI);
+      /* eslint-enable no-new */
+    }).toThrow();
+  });
+
+  describe('a new Infibeam Site', () => {
+    let site;
+
+    beforeEach(() => {
+      site = new InfibeamSite(VALID_URI);
+    });
+
+    it('should exist', () => {
+      expect(site).toBeDefined();
+    });
+
+    it('should return the same URI for getURIForPageData()', () => {
+      expect(site.getURIForPageData()).toEqual(VALID_URI);
+    });
+
+    it('should return false for isJSON()', () => {
+      expect(site.isJSON()).toBeFalsy();
+    });
+
+    describe('with a populated page', () => {
+      let $;
+      let bad$;
+      let price;
+      let category;
+      let name;
+
+      beforeEach(() => {
+        price = 9444;
+        category = siteUtils.categories.ELECTRONICS;
+        name = 'Sansui SJX22FB Full HD LED TV';
+
+        // TODO provide content for unit test here!
+        $ = cheerio.load('<div id="price-after-discount">' +
+          '<span style="font-family: rupee">R </span><span class="price">$9,444</span></div>' +
```

Comment:
This maybe my confusion, but does the page display both `R` and `$` in the price?  Or is it simply `R 9,444`?